client/src/pages/EditDetails/index.js

```diff
@@ -219,24 +264,36 @@ const EditDetails = () => {
             {t('editDetails.title', lang)}
           </T.H1>
         </Col>{' '}
-        <Col w={[4, 6, 6]}>
+        <Col w={[4, 12, 6]}>
           <T.P isSmall color="neutralDark" mt={4}>
             {t('editDetails.subtitle', lang)}
           </T.P>
         </Col>
       </Row>
       <Row mt={6}>
-        <Col w={[4, 4, 4]}>
+        <Col w={[4, 6, 4]}>
           <I.BasicInput
-            label="Name"
+            label="First name"
             type="text"
-            placeholder="Type name here..."
-            value={name}
-            handleChange={(name) => setFormData({ name })}
-            error={validationErrs?.name}
+            placeholder="Type first name here..."
+            value={firstName}
+            handleChange={(firstName) => setFormData({ firstName })}
+            error={validationErrs?.firstName}
           />
         </Col>
-        <Col w={[4, 4, 4]} mt={isMobile ? 6 : 0}>
+        <Col w={[4, 6, 4]}>
+          <I.BasicInput
+            label="Last name"
+            type="text"
+            placeholder="Type last name here..."
+            value={lastName}
+            handleChange={(lastName) => setFormData({ lastName })}
+            error={validationErrs?.lastName}
+          />
```

Comment:
The design has only one input for name, if you want to send first & last name you can take the `name` value and split it into two values, and send them as separate values to the backend

Reply:
Resolving this as it appears that the design isn't up to date